tests/test_processor_remote_tiles_westport/test_processor_remote_tiles_westport.py

```diff
@@ -105,7 +105,7 @@ def test_correct_dataset(self):
         dataset_dir = self.cache_dir / self.DATASET
 
         # check the right dataset is downloaded - self.DATASET
-        self.assertEqual(len(list(self.cache_dir.glob('*/**'))), 1,
+        self.assertEqual(len(list(self.cache_dir.glob('*/**'))), 2,  # The extents.json file as well
                          f"There should only be one dataset named {self.DATASET} instead there are " +
                          f"{len(list(self.cache_dir.glob('*/**')))} list {list(self.cache_dir.glob('*/**'))}")
```

Comment:
This assertion error message could be confusing since the expected is 2 but the message states 1